Cargo.toml

```diff
@@ -14,6 +14,14 @@ exclude = [
     "proto-compiler"
 ]
 
+[profile.dev]
+# Using line tables only for debug info speeds up builds a bunch
```

Comment:
Hmm I don't understand this comment.

Reply:
I just added some similar changes to `modelator`. If this is just for CI, it should be enough to set [CARGO_PROFILE_DEV_DEBUG](https://doc.rust-lang.org/cargo/reference/config.html#profilenamedebug) on workflow environment.

Reply:
This means that the debug information embedded in the binary will only include information about line numbers and function names rather than the full debug information which is typically used by debugger such as gdb or lldb. This has reported to speed up the builds by up to 15%. On the downside, one now has to change this setting back to 2 if they want to use a debugger and do a full rebuild.

That said, I like @rnbguy's suggestion to use an env variable to set this option on the CI only.

Reply:
Thank you for the explanation.